setup.cfg

```diff
@@ -57,7 +57,10 @@ sort = Miss
 include =
     coalib/*
 exclude_lines =
-    pragma: no ?cover
     pragma ${PLATFORM_SYSTEM}: no cover
     pragma ${OS_NAME}: no cover
     pragma Python [0-9.,]*${PYTHON_VERSION}[0-9.,]*: no cover
+    raise RuntimeError.*Unreachable code
```

Comment:
maybe we should put those two changes into a separate commit? So first we try to improve current pragmas, and in the end we remove the generic one?